javascript/ql/src/semmle/javascript/dataflow/TrackedNodes.qll

```diff
@@ -1,4 +1,6 @@
 /**
+ * DEPRECATED: Use `TypeTracking.qll` or a `DataFlow::Configuration` from `Configuration.qll` instead.
```

Comment:
Lets avoid encouraging spurious introductions of `Configuration`. The curious programmers will find that solution by themselves. I think we should show an example of how one can address this deprecation. I have started below.

```suggestion
 * DEPRECATED: Use `TypeTracking.qll` instead.
 * 
 * The following `TrackedNode` usage is less expressive than the type tracking usage below.
 *
 * ` ` ` <-- remove spaces
 * class MyTrackedNode extends TrackedNode {
 *    MyTrackedNode() { isInteresting(this) }
 * }
 *
 * DataFlow::Node getMyTrackedNodeLocation(MyTrackedNode n) {
 *   n.flowsTo(result)
 * }
 *
 ...


```